src/test/java/com/ncc/aif/TestUtils.java

```diff
@@ -0,0 +1,466 @@
+     */
+    NistTestUtils(String annotationNamespace, ValidateAIF validator, boolean forceDump) {
+        super(annotationNamespace, validator, forceDump);
+    }
+
+    /**
+     * Call before each test.  Returns a new, empty model with standard AIF namespaces.
+     *
+     * @return a new model with which to start a test; caller may wish to add prefixes for the ontology and annotation
+     */
+    Model startNewTest() {
+        Model model = super.startNewTest();
+        // NIST tests always need type assertion justifications
+        addSourceDocumentToJustification(super.getTypeAssertionJustification(), getDocumentName());
+        return model;
+    }
+
+    /**
+     * Makes and returns a valid NIST-restricted entity of the specified type and its cluster.
+     *
+     * @param type entity type
+     * @return a key-value Pair of the entity Resource (key) and its associated cluster Resource (value)
+     */
+    Pair<Resource, Resource> makeNistEntity(Resource type) {
+        Resource entity = AIFUtils.makeEntity(model, getEntityUri(), system);
+        markJustification(addType(entity, type), getTypeAssertionJustification());
+        Resource entityCluster = makeClusterWithPrototype(model, getClusterUri(), entity, system);
+        return new Pair<>(entity, entityCluster);
+    }
+
+    /**
+     * Makes and returns a valid NIST-restricted event of the specified type and its cluster.
+     *
+     * @param type event type
+     * @return a key-value Pair of the event Resource (key) and its associated cluster Resource (value)
+     */
+    Pair<Resource, Resource> makeNistEvent(Resource type) {
+        Resource event = AIFUtils.makeEvent(model, getEventUri(), system);
+        markJustification(addType(event, type), getTypeAssertionJustification());
+        Resource entityCluster = makeClusterWithPrototype(model, getClusterUri(), event, system);
+        return new Pair<>(event, entityCluster);
+    }
+
+    /**
+     * Makes and returns a valid NIST-restricted relation of the specified type and its cluster.
+     *
+     * @param type relation type
+     * @return a key-value Pair of the relation Resource (key) and its associated cluster Resource (value)
+     */
+    Pair<Resource, Resource> makeNistRelation(Resource type) {
+        Resource relation = AIFUtils.makeRelation(model, getEventUri(), system);
+        markJustification(addType(relation, type), getTypeAssertionJustification());
+        Resource relationCluster = makeClusterWithPrototype(model, getClusterUri(), relation, system);
+        return new Pair<>(relation, relationCluster);
+    }
+
+    /**
+     * Makes and returns a valid NIST-restricted hypothesis involving the specified resource(s).
+     *
+     * @param resources A set of entities, relations, and arguments that contribute to the hypothesis
+     */
+    Resource makeNistHypothesis(Resource... resources) {
```

Comment:
This should go in the NistHypothesisTestUtils class.  Is there some reason why it is here instead?

Reply:
Nope, I just created the NistHypothesisTestUtils class at the last moment and missed moving this one.